### PR TITLE
chore(skilavottord): cleanup charts

### PR DESCRIPTION
**Pull request by darrikonn**
# Cleanup charts

## What

Cleanup charts and update README.md

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by miamiultras — APPROVED** [review]

**Review by qdd1 — APPROVED** [review]

File: apps/skilavottord/README.md [hunk 1]
```diff
@@ -32,6 +33,18 @@ See role description further down.
...
+### Setup
+
+Please fetch environment secret variables using the
+`yarn get-secrets skilavottord-ws` command.
+
+Additionally add the following environment variables to `.env.secret` in the
+project's root:
+
+```bash
+export NEXTAUTH_URL=http://localhost:4200/app/skilavottord/api/auth
```
Comment (dabbeg) [inline apps/skilavottord/README.md:45]: Is this needed? Seems like the default value in the environments is the same.

**Review by dabbeg — APPROVED** [review]
Nice

File: apps/skilavottord/README.md (hunk 1 above)
Comment (darrikonn) [reply apps/skilavottord/README.md:45]: next-auth requires this environment variable to be set.

Comment (dabbeg) [reply apps/skilavottord/README.md:45]: Huh, even though it is set here: [https://github.com/island-is/island.is/pull/6119/files#diff-907a615c9ccc2e81a8b2f74817e2c8c930e0fd17f481cac80379f51cef3a328cR9](https://github.com/island-is/island.is/pull/6119/files#diff-907a615c9ccc2e81a8b2f74817e2c8c930e0fd17f481cac80379f51cef3a328cR9)?

Comment (darrikonn) [reply apps/skilavottord/README.md:45]: yuup

Comment (dabbeg) [reply apps/skilavottord/README.md:45]: :exploding_head:

**Pull request merged**